winlogtimeline/ui/ui.py

```diff
@@ -521,14 +528,15 @@ def import_button_function(self, event=None):
         wizard.grab_set()
 
     @enable_disable_wrapper(lambda *args: args[0].master)
-    def format_button_function(self, event=None):
+    def export_button_function(self, event=None):
         """
         TODO: Determine whether or not this function and associated button are necessary.
         :param event:
         :return:
         """
-        self.master.status_bar.status.config(text='"Format" button pressed.')
-        return
+        print(self.master.current_project)
+        wizard = ExportWindow(self, self.master.current_project)
+        wizard.grab_set()
 
     def __enable__(self):
```

Comment:
Please add the export menu option to the __enable__ and __disable__ of the menu. The feature doesn't work on macs, but it still looks cleaner than a status bar message on Windows.

Reply:
Ah good note. I didn't see that. That's been fixed.